[2022-07-05 07:21:35,830] {taskinstance.py:1037} INFO - Dependencies all met for <TaskInstance: uk_dag_test_1.kimuksung_test scheduled__2022-07-05T01:00:00+00:00 [queued]>
[2022-07-05 07:21:35,867] {taskinstance.py:1037} INFO - Dependencies all met for <TaskInstance: uk_dag_test_1.kimuksung_test scheduled__2022-07-05T01:00:00+00:00 [queued]>
[2022-07-05 07:21:35,867] {taskinstance.py:1243} INFO - 
--------------------------------------------------------------------------------
[2022-07-05 07:21:35,868] {taskinstance.py:1244} INFO - Starting attempt 1 of 1
[2022-07-05 07:21:35,868] {taskinstance.py:1245} INFO - 
--------------------------------------------------------------------------------
[2022-07-05 07:21:35,917] {taskinstance.py:1264} INFO - Executing <Task(PythonOperator): kimuksung_test> on 2022-07-05 01:00:00+00:00
[2022-07-05 07:21:35,931] {standard_task_runner.py:52} INFO - Started process 602 to run task
[2022-07-05 07:21:35,939] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'uk_dag_test_1', 'kimuksung_test', 'scheduled__2022-07-05T01:00:00+00:00', '--job-id', '254', '--raw', '--subdir', 'DAGS_FOLDER/test.py', '--cfg-path', '/tmp/tmp0ww8g1j0', '--error-file', '/tmp/tmp6yvhlqzi']
[2022-07-05 07:21:35,939] {standard_task_runner.py:77} INFO - Job 254: Subtask kimuksung_test
[2022-07-05 07:21:36,167] {logging_mixin.py:109} INFO - Running <TaskInstance: uk_dag_test_1.kimuksung_test scheduled__2022-07-05T01:00:00+00:00 [running]> on host 9514ca4c721b
[2022-07-05 07:21:36,333] {taskinstance.py:1431} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=uk_dag_test_1
AIRFLOW_CTX_TASK_ID=kimuksung_test
AIRFLOW_CTX_EXECUTION_DATE=2022-07-05T01:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-07-05T01:00:00+00:00
[2022-07-05 07:21:36,335] {logging_mixin.py:109} INFO - test function start
[2022-07-05 07:21:36,335] {taskinstance.py:1718} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/taskinstance.py", line 1334, in _run_raw_task
    self._execute_task_with_callbacks(context)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/taskinstance.py", line 1460, in _execute_task_with_callbacks
    result = self._execute_task(context, self.task)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/taskinstance.py", line 1516, in _execute_task
    result = execute_callable(context=context)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/python.py", line 174, in execute
    return_value = self.execute_callable()
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/python.py", line 188, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
  File "/opt/airflow/dags/test.py", line 21, in test
    parameter=context["paramas"]["parameter1"]
KeyError: 'paramas'
[2022-07-05 07:21:36,370] {taskinstance.py:1282} INFO - Marking task as FAILED. dag_id=uk_dag_test_1, task_id=kimuksung_test, execution_date=20220705T010000, start_date=20220705T072135, end_date=20220705T072136
[2022-07-05 07:21:36,415] {standard_task_runner.py:92} ERROR - Failed to execute job 254 for task kimuksung_test
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/task/task_runner/standard_task_runner.py", line 85, in _start_by_fork
    args.func(args, dag=self.dag)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/cli/cli_parser.py", line 48, in command
    return func(*args, **kwargs)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/utils/cli.py", line 92, in wrapper
    return f(*args, **kwargs)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/cli/commands/task_command.py", line 298, in task_run
    _run_task_by_selected_method(args, dag, ti)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/cli/commands/task_command.py", line 107, in _run_task_by_selected_method
    _run_raw_task(args, ti)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/cli/commands/task_command.py", line 184, in _run_raw_task
    error_file=args.error_file,
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/utils/session.py", line 70, in wrapper
    return func(*args, session=session, **kwargs)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/taskinstance.py", line 1334, in _run_raw_task
    self._execute_task_with_callbacks(context)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/taskinstance.py", line 1460, in _execute_task_with_callbacks
    result = self._execute_task(context, self.task)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/taskinstance.py", line 1516, in _execute_task
    result = execute_callable(context=context)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/python.py", line 174, in execute
    return_value = self.execute_callable()
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/python.py", line 188, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
  File "/opt/airflow/dags/test.py", line 21, in test
    parameter=context["paramas"]["parameter1"]
KeyError: 'paramas'
[2022-07-05 07:21:36,450] {local_task_job.py:154} INFO - Task exited with return code 1
[2022-07-05 07:21:36,524] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2022-07-05 07:25:49,540] {taskinstance.py:1037} INFO - Dependencies all met for <TaskInstance: uk_dag_test_1.kimuksung_test scheduled__2022-07-05T01:00:00+00:00 [queued]>
[2022-07-05 07:25:49,594] {taskinstance.py:1037} INFO - Dependencies all met for <TaskInstance: uk_dag_test_1.kimuksung_test scheduled__2022-07-05T01:00:00+00:00 [queued]>
[2022-07-05 07:25:49,594] {taskinstance.py:1243} INFO - 
--------------------------------------------------------------------------------
[2022-07-05 07:25:49,594] {taskinstance.py:1244} INFO - Starting attempt 1 of 1
[2022-07-05 07:25:49,594] {taskinstance.py:1245} INFO - 
--------------------------------------------------------------------------------
[2022-07-05 07:25:49,651] {taskinstance.py:1264} INFO - Executing <Task(PythonOperator): kimuksung_test> on 2022-07-05 01:00:00+00:00
[2022-07-05 07:25:49,659] {standard_task_runner.py:52} INFO - Started process 793 to run task
[2022-07-05 07:25:49,669] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'uk_dag_test_1', 'kimuksung_test', 'scheduled__2022-07-05T01:00:00+00:00', '--job-id', '285', '--raw', '--subdir', 'DAGS_FOLDER/test.py', '--cfg-path', '/tmp/tmp6g3ggix2', '--error-file', '/tmp/tmpwgrbpejp']
[2022-07-05 07:25:49,670] {standard_task_runner.py:77} INFO - Job 285: Subtask kimuksung_test
[2022-07-05 07:25:49,920] {logging_mixin.py:109} INFO - Running <TaskInstance: uk_dag_test_1.kimuksung_test scheduled__2022-07-05T01:00:00+00:00 [running]> on host 9514ca4c721b
[2022-07-05 07:25:50,304] {taskinstance.py:1431} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=uk_dag_test_1
AIRFLOW_CTX_TASK_ID=kimuksung_test
AIRFLOW_CTX_EXECUTION_DATE=2022-07-05T01:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-07-05T01:00:00+00:00
[2022-07-05 07:25:50,306] {logging_mixin.py:109} INFO - test function start
[2022-07-05 07:25:50,306] {test.py:22} INFO - pubg
[2022-07-05 07:25:50,306] {python.py:175} INFO - Done. Returned value was: test function clear
[2022-07-05 07:25:50,456] {taskinstance.py:1282} INFO - Marking task as SUCCESS. dag_id=uk_dag_test_1, task_id=kimuksung_test, execution_date=20220705T010000, start_date=20220705T072549, end_date=20220705T072550
[2022-07-05 07:25:50,535] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-07-05 07:25:50,865] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2022-07-05 08:06:25,893] {taskinstance.py:1037} INFO - Dependencies all met for <TaskInstance: uk_dag_test_1.kimuksung_test scheduled__2022-07-05T01:00:00+00:00 [queued]>
[2022-07-05 08:06:25,983] {taskinstance.py:1037} INFO - Dependencies all met for <TaskInstance: uk_dag_test_1.kimuksung_test scheduled__2022-07-05T01:00:00+00:00 [queued]>
[2022-07-05 08:06:25,983] {taskinstance.py:1243} INFO - 
--------------------------------------------------------------------------------
[2022-07-05 08:06:25,984] {taskinstance.py:1244} INFO - Starting attempt 1 of 1
[2022-07-05 08:06:25,985] {taskinstance.py:1245} INFO - 
--------------------------------------------------------------------------------
[2022-07-05 08:06:26,109] {taskinstance.py:1264} INFO - Executing <Task(PythonOperator): kimuksung_test> on 2022-07-05 01:00:00+00:00
[2022-07-05 08:06:26,195] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'uk_dag_test_1', 'kimuksung_test', 'scheduled__2022-07-05T01:00:00+00:00', '--job-id', '341', '--raw', '--subdir', 'DAGS_FOLDER/test.py', '--cfg-path', '/tmp/tmppf4o2bcw', '--error-file', '/tmp/tmp9shc4yw_']
[2022-07-05 08:06:26,196] {standard_task_runner.py:77} INFO - Job 341: Subtask kimuksung_test
[2022-07-05 08:06:26,133] {standard_task_runner.py:52} INFO - Started process 2182 to run task
[2022-07-05 08:06:26,956] {logging_mixin.py:109} INFO - Running <TaskInstance: uk_dag_test_1.kimuksung_test scheduled__2022-07-05T01:00:00+00:00 [running]> on host 9514ca4c721b
[2022-07-05 08:06:27,417] {taskinstance.py:1431} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=uk_dag_test_1
AIRFLOW_CTX_TASK_ID=kimuksung_test
AIRFLOW_CTX_EXECUTION_DATE=2022-07-05T01:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-07-05T01:00:00+00:00
[2022-07-05 08:06:27,418] {logging_mixin.py:109} INFO - test function start
[2022-07-05 08:06:27,418] {test.py:22} INFO - pubg
[2022-07-05 08:06:27,419] {python.py:175} INFO - Done. Returned value was: test function clear
[2022-07-05 08:06:27,555] {taskinstance.py:1282} INFO - Marking task as SUCCESS. dag_id=uk_dag_test_1, task_id=kimuksung_test, execution_date=20220705T010000, start_date=20220705T080625, end_date=20220705T080627
[2022-07-05 08:06:27,688] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-07-05 08:06:28,056] {local_task_job.py:264} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2022-07-05 08:11:39,271] {taskinstance.py:1037} INFO - Dependencies all met for <TaskInstance: uk_dag_test_1.kimuksung_test scheduled__2022-07-05T01:00:00+00:00 [queued]>
[2022-07-05 08:11:39,407] {taskinstance.py:1037} INFO - Dependencies all met for <TaskInstance: uk_dag_test_1.kimuksung_test scheduled__2022-07-05T01:00:00+00:00 [queued]>
[2022-07-05 08:11:39,415] {taskinstance.py:1243} INFO - 
--------------------------------------------------------------------------------
[2022-07-05 08:11:39,415] {taskinstance.py:1244} INFO - Starting attempt 1 of 1
[2022-07-05 08:11:39,415] {taskinstance.py:1245} INFO - 
--------------------------------------------------------------------------------
[2022-07-05 08:11:39,614] {taskinstance.py:1264} INFO - Executing <Task(PythonOperator): kimuksung_test> on 2022-07-05 01:00:00+00:00
[2022-07-05 08:11:39,655] {standard_task_runner.py:52} INFO - Started process 2486 to run task
[2022-07-05 08:11:39,693] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'uk_dag_test_1', 'kimuksung_test', 'scheduled__2022-07-05T01:00:00+00:00', '--job-id', '406', '--raw', '--subdir', 'DAGS_FOLDER/test.py', '--cfg-path', '/tmp/tmp1kkh2wdp', '--error-file', '/tmp/tmpy8awjsgz']
[2022-07-05 08:11:39,694] {standard_task_runner.py:77} INFO - Job 406: Subtask kimuksung_test
[2022-07-05 08:11:40,229] {logging_mixin.py:109} INFO - Running <TaskInstance: uk_dag_test_1.kimuksung_test scheduled__2022-07-05T01:00:00+00:00 [running]> on host 9514ca4c721b
[2022-07-05 08:11:40,672] {taskinstance.py:1431} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=uk_dag_test_1
AIRFLOW_CTX_TASK_ID=kimuksung_test
AIRFLOW_CTX_EXECUTION_DATE=2022-07-05T01:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-07-05T01:00:00+00:00
[2022-07-05 08:11:40,674] {logging_mixin.py:109} INFO - test function start
[2022-07-05 08:11:40,675] {test.py:22} INFO - pubg
[2022-07-05 08:11:40,676] {python.py:175} INFO - Done. Returned value was: pubg
[2022-07-05 08:11:40,811] {taskinstance.py:1282} INFO - Marking task as SUCCESS. dag_id=uk_dag_test_1, task_id=kimuksung_test, execution_date=20220705T010000, start_date=20220705T081139, end_date=20220705T081140
[2022-07-05 08:11:41,022] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-07-05 08:11:41,308] {local_task_job.py:264} INFO - 1 downstream tasks scheduled from follow-on schedule check
